english/Golf/Golf.md

```diff
@@ -0,0 +1,9 @@
+You check out your drawes, yes, they are accessible.
+You go ahead and put your favorite footjoy shoes, red, and you match it with black pants red nike polo and a red and black taylor made cup. Now with who are you going to golf?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.